packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -446,21 +446,27 @@ export const components = ({
           }),
 
           ...(ownerState.variant === "secondary" && {
-            backgroundColor: odysseyTokens.HueBlue100,
-            color: odysseyTokens.PalettePrimaryDark,
+            backgroundColor: "transparent",
+            borderColor: odysseyTokens.HueNeutral300,
+            color: odysseyTokens.TypographyColorBody,
 
             "&:hover": {
-              backgroundColor: odysseyTokens.HueBlue200,
-              color: odysseyTokens.HueBlue800,
+              backgroundColor: odysseyTokens.HueNeutral100,
             },
 
             "&:active": {
-              backgroundColor: odysseyTokens.PalettePrimaryLight,
-              color: odysseyTokens.HueBlue800,
+              backgroundColor: "transparent",
+              borderColor: odysseyTokens.BorderColorPrimaryControl,
+              color: odysseyTokens.TypographyColorAction,
+            },
+
+            "&:focus": {
+              backgroundColor: odysseyTokens.HueNeutralWhite,
             },
```

Comment:
This forces the background to be white when it's focused _even when it's hovered over_